public/static/docs/command-reference/update.md

```diff
@@ -69,3 +77,43 @@ stable.
 > Note that `dvc update` updates the `rev_lock` field of the corresponding
 > [DVC-file](/doc/user-guide/dvc-file-format) (when there are changes to bring
 > in).
+
+## Example: Updating imported artifacts to a specified revision
+
+Let's import a data artifact from an older commit from our
+[get started example repo](https://github.com/iterative/example-get-started) at
+first:
+
+```dvc
+$ dvc import --rev 72e0f1 git@github.com:iterative/example-get-started model.pkl
+Importing 'model.pkl (git@github.com:iterative/example-get-started)'
+-> 'model.pkl'
+```
+
+After this, the import stage (DVC-file) `model.pkl.dvc` is created.
+
+Let's try to run `dvc update` on the given stage file, and see what happens.
+
+```dvc
+$ dvc update model.pkl.dvc
+```
+
+There was no output at all, meaning, the `model.pkl` file was not updated. This
+is because, we tied the import stage with a `rev` that never changes (i.e.
+commit hash never changes). Therefore, it was not updated.
+
+Let's try to update it to the latest version by specifying revision to `master`
+branch.
+
+```dvc
+$ dvc update --rev master model.pkl.dvc
+WARNING: DVC-file 'model.pkl.dvc' changed.
```

Comment:
same here - let's not include outputs

Reply:
How I wrote the whole examples here, I feel it's important to show the output (that no output is there if there's no any updates). But, we won't need this if we change the example.

Reply:
👍 up to you on this